drafts/zip-str4d-multipart-memos.rst

```diff
@@ -0,0 +1,171 @@
+::
+
+  ZIP: XXX
+  Title: Multipart Memos
+  Owners: Jack Grigg <jack@electriccoin.co>
+  Status: Draft
+  Category: Standards
+  Created: 2019-06-27
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST NOT", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be
+interpreted as described in RFC 2119. [#RFC2119]_
+
+
+Abstract
+========
+
+This proposal defines a mechanism for encoding memos to recipients that are larger than a
+single encrypted memo field.
+
+
+Motivation
+==========
+
+Shielded outputs within transactions contain an encrypted 512-byte memo field, decryptable
+only by the recipient that the output is for. This is sufficient for many use cases, such
+as disclosing information about the sender to the recipient. However, there are potential
+use cases that may require more than 512 bytes of data to be transferred alongside the
+transaction payment to the recipient.
+
+It is possible to send multiple outputs to the same recipient address within a single
+transaction, and it is also possible for some or all of these outputs to have zero value
+(dummy outputs). This can be leveraged to convey more information to the recipient than
```

Comment:
```suggestion
transaction, and it is also possible for some or all of these shielded outputs to have zero value
(dummy shielded outputs). This can be leveraged to convey more information to the recipient than
```